elastic/kibana
# [Infrastructure UI] Telemetry: Host view flyout - Add / Remove filter

Closes [#155415](https://github.com/elastic/kibana/issues/155415)
## Summary

This PR adds telemetry for the flyout add/remove filter events

# Testing

- Open the hosts view flyout, select the metadata tab and add/remove a filter from there
   - Check in the network tab - there should be a request to the telemetry cluster including the events: 
<img width="2126" alt="image" src="https://user-images.githubusercontent.com/14139027/233965729-58fd6266-7851-48e5-acf0-e395b4fd7ebe.png">

   - It should be also visible in [FS](https://app.fullstory.com/ui/1397FY/home) under API events (for some reason I don't see the events made from my machine in FS at all so I couldn't check it there)
